addons/project/static/src/components/project_right_side_panel/project_right_side_panel.xml

```diff
@@ -37,17 +37,22 @@
             </ProjectRightSidePanelSection>
             <ProjectRightSidePanelSection
                 name="'profitability'"
-                show="showProjectProfitability"
+                show="!!this.state.data.profitability_items"
             >
                 <t t-set-slot="title" owl="1">
                     Profitability
                 </t>
-                <ProjectProfitability
-                    data="state.data.profitability_items"
-                    labels="state.data.profitability_labels"
-                    formatMonetary="formatMonetary.bind(this)"
-                    onClick="(params) => this.onProjectActionClick(params)"
-                />
+                <span t-if="showProjectProfitabilityHelper" class="o_default_body">
```

Comment:
```suggestion
                <span t-if="showProjectProfitability" class="o_default_body">
```
Oops, but now that we use it to determine if we will show ProjectProfitability, we should rename it as it was before :see_no_evil: